_posts/2022-11-18-optimizing-production-pytorch-performance-with-graph-transformations.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog_detail
+title: "Optimizing Production PyTorch Models’ Performance with Graph Transformations"
+author: Jade Nie, CK Luk, Xiaodong Wang, Jackie (Jiaqi) Xu
+featured-img: "assets/images/blog1-3b.png"
```

Comment:
I don't see this image asset in the folder. Should this be 'assets/images/blog1-fig-3b.png'?